CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Version 0.4.2 (2023-07-04)
+
+Updating dependencys, specifically axoasset, to remove OpenSSL dependency.

```

Comment:
```suggestion
Updating dependencies, specifically axoasset, to remove OpenSSL dependency.

```

Reply:
lol i really should finish my coffee before doing this. thank you